src/main/resources/org/jenkinsci/plugins/workflow/support/steps/build/RunWrapper/help.html

```diff
@@ -0,0 +1,16 @@
+<dl>
+    <dt><code>number</code></dt><dd>build number (integer)</dd>
+    <dt><code>result</code></dt><dd>typically <code>SUCCESS</code>, <code>UNSTABLE</code>, or <code>FAILURE</code> (<em>may</em> be null for an ongoing build)</dd>
+    <dt><code>displayName</code></dt><dd>normally <code>#123</code> but sometimes set to, e.g., an SCM commit identifier</dd>
+    <dt><code>description</code></dt><dd>additional information about the build</dd>
+    <dt><code>id</code></dt><dd>normally <code>number</code> as a string</dd>
+    <dt><code>timeInMillis</code></dt><dd>time since the epoch when the build was scheduled</dd>
+    <dt><code>startTimeInMillis</code></dt><dd>time since the epoch when the build started running</dd>
+    <dt><code>duration</code></dt><dd>duration of the build in milliseconds</dd>
+    <dt><code>previousBuild</code></dt><dd>another similar object, or null</dd>
+    <dt><code>nextBuild</code></dt><dd>similarly</dd>
+    <dt><code>absoluteUrl</code></dt><dd>URL of build index page</dd>
+    <dt><code>buildVariables</code></dt><dd>for a non-Pipeline downstream build, offers access to a map of defined build variables; for a Pipeline downstream build, any variables set globally on <code>env</code></dd>
+    <dt><code>changeSets</code></dt><dd>a list of <a href="http://javadoc.jenkins-ci.org/hudson/scm/ChangeLogSet.html" target="_blank">changesets</a> coming from distinct SCM checkouts; each has a <code>kind</code> and is a list of commits; each commit has a <code>commitId</code>, <code>timestamp</code>, <code>msg</code>, <code>author</code>, and <code>affectedFiles</code> each of which has an <code>editType</code> and <code>path</code>; the value will not generally be <code>Serializable</code> so you may only access it inside a method marked <code>@NonCPS</code></dd>
+    <dt><code>rawBuild</code></dt><dd>a <code>hudson.model.Run</code> with <a href="http://javadoc.jenkins-ci.org/hudson/model/Run.html" target="_blank">further APIs</a>, only for trusted libraries or administrator-approved scripts outside the sandbox; the value will not be <code>Serializable</code> so you may only access it inside a method marked <code>@NonCPS</code></dd>
```

Comment:
The last two lines edited a bit to emphasize [`workflow-cps` 2.14](https://wiki.jenkins-ci.org/display/JENKINS/Pipeline+Groovy+Plugin#PipelineGroovyPlugin-2.14%28Sep07%2C2016%29).